test/screenshot/report/report.js

```diff
@@ -127,15 +148,6 @@ window.mdc.reportUi = (() => {
     }
 
     collapseImages() {
-      this.collapseNone();
```

Comment:
Can you explain why this was moved?

Reply:
Added a bullet point to the PR description 😄 

I found it really annoying that the "Collapse Images" button did more than just collapse images. It also expanded/collapsed other `<details>` sections, which was unexpected and generally unwanted behavior (at least to me).